VENCore.podspec

```diff
@@ -1,19 +1,14 @@
 Pod::Spec.new do |s|
-  s.name         = "VENCore"
-  s.version      = "3.1.1"
-  s.summary      = "Core Venmo client library"
-  s.description  = <<-DESC
-                   Core iOS client library for the Venmo api
-                   DESC
-
-  s.homepage     = "https://github.com/venmo/VENCore"
+  s.name         = 'VENCore'
+  s.version      = '3.1.2'
+  s.summary      = 'Core Venmo client library'
+  s.description  = 'Core iOS client library for the Venmo api'
+  s.homepage     = 'https://github.com/venmo/VENCore'
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
-  s.author       = { "Venmo" => "ios@venmo.com" }
+  s.author       = { 'Venmo' => 'ios@venmo.com' }
   s.platform     = :ios, '6.0'
-  s.source       = { :git => "https://github.com/venmo/VENCore.git",
-                     :tag => "v#{s.version}"
-  }
+  s.source       = { :git => 'https://github.com/venmo/VENCore.git',
+                     :tag => "v#{s.version}" }
   s.source_files = 'VENCore/**/*.{h,m}'
-  s.dependency 'CMDQueryStringSerialization', '~>0.2.0'
-  s.requires_arc = true
+  s.dependency 'CMDQueryStringSerialization'
```

Comment:
By removing the version here, we're saying that any version of `CMDQueryStringSerialization` should work with this repo. If a user declares `pod 'CMDQueryStringSerialization', '~> 0.2.0'` in their Podfile and include this latest version of VENCore that includes this change with no version restriction, there may be some issues in resolving the appropriate dependencies.

I'm really not for removing version requirements in podspecs, although it may be fine in Podfiles.